sre/roles/tools/tasks/reinit_chaos_mesh.yaml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Get all custom resource definitions - CRDs
+  kubernetes.core.k8s_info:
+    kubeconfig: "{{ tools_cluster.kubeconfig }}"
+    api_version: apiextensions.k8s.io/v1
+    kind: CustomResourceDefinition
+  register: all_crds
+
+- name: Extract chaos-mesh resource types - CRDs
+  set_fact:
+    chaos_mesh_crds: "{{ all_crds.resources | selectattr('spec.group', 'search', 'chaos-mesh') | list }}"
+
+- name: Get all instances of each chaos-mesh resource type
+  kubernetes.core.k8s_info:
+    kubeconfig: "{{ tools_cluster.kubeconfig }}"
+    api_version: "{{ item.spec.group }}/{{ item.spec.versions[0].name }}"
+    kind: "{{ item.spec.names.kind }}"
+  loop: "{{ chaos_mesh_crds }}"
```

Comment:
Maybe something more like this so the terminal is not filled with a whole CRD per loop?

https://github.com/IBM/ITBench-Scenarios/blob/355b1e832d9609c364056cc84d848bf10c5621bb/sre/roles/tools/tasks/uninstall_prometheus.yaml#L31